dev/tests/integration/testsuite/Magento/Sales/Api/ShipmentTrackRepositoryInterfaceTest.php

```diff
@@ -41,14 +41,17 @@ public function testGetList()
         $filter3 = $filterBuilder->setField(ShipmentTrackInterface::TRACK_NUMBER)
             ->setValue('track number 4')
             ->create();
-        $filter4 = $filterBuilder->setField(ShipmentTrackInterface::CARRIER_CODE)
-            ->setValue('carrier code 5')
+        $filter4 = $filterBuilder->setField(ShipmentTrackInterface::TRACK_URL)
```

Comment:
You added new search criteria but didn't change the test's fixture. Does this test work as expected?

Reply:
Hi, I've updated the tests again, including the fixtures, please check again

Reply:
Confirmed.